src/visualizers/VisualizerTurtle.ts

```diff
@@ -130,7 +130,7 @@ class VisualizerTurtle extends VisualizerDefault implements VisualizerInterface
 	}
 
 	draw() {
-		const currElement = this.seq.getElement(this.currentIndex++);
+        const currElement = Number(this.seq.getElement(this.currentIndex++));
```

Comment:
I don't understand why this isn't indented.

Reply:
When I got to this code file, the indentation was all over the map: some authors had used one "TAB" character (Ctrl-I) per indent level, others had used four spaces per indent level. If your editor had tabs set at four spaces, then it would look OK, but not all editors do. I wanted to minimize touching of the code, so I left this crazy situation alone. I will change this function to use all spaces, which I recommend as the convention because tab stops do differ from environment to environment, but a space is always a space.

Reply:
Well, hopefully all of this weirdness with the indentation will go away with the addition of the code formatter!